ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/AndroidApplicationContext.kt

```diff
@@ -18,8 +18,8 @@ internal class AndroidApplicationContext(private val applicationContext: Context
         return applicationContext.applicationInfo.loadLabel(applicationContext.packageManager).toString()
     }
 
-    override fun getString(@StringRes resId: Int): String {
-        return applicationContext.resources.getString(resId)
+    override fun getString(@StringRes resId: Int, vararg formatArgs: Any): String {
+        return applicationContext.resources.getString(resId, *formatArgs)
```

Comment:
Oh nice 👍🏻

Reply:
I didn't know about that `*` but without it, kotlin pases an object that contains another object instead of the object itself. It was hard to realize lol